project.clj

```diff
@@ -33,12 +35,13 @@
 
   :aliases {"kaocha" ["run" "-m" "kaocha.runner"]}
   :aot [jackdaw.serdes.edn2 jackdaw.serdes.fressian jackdaw.serdes.fn-impl]
-  :plugins [[me.arrdem/lein-git-version "2.0.8"]]
+  :plugins [[me.arrdem/lein-git-version "2.0.8"]
+            [com.github.clj-kondo/lein-clj-kondo "0.2.5"]]
 
   :git-version
   {:status-to-version
    (fn [{:keys [tag branch ahead? dirty?] :as git}]
-     (if (and tag (not ahead?) (not dirty?))
+     (if (and tag (not ahead?) (not dirty?) (= "master" branch))
```

Comment:
Should this be handled with trigger filters in the workflow def?

Reply:
Added to make more explicit that all non-master builds are marked snapshot; it should be handled already by the current setup, happy to remove it it adds confusion.